src/project/mesa3d_desktop_intel.rs

```diff
@@ -199,7 +183,7 @@ cc_defaults {{
         };
         let module = relative_install(module);
 
-        let mut libs = Vec::new();
```

Comment:
If you add them unconditionally, maybe you want them to also be in the default like `soc_specific`

Reply:
Okay, thanks!